### ISSUE TITLE
Drop the raw atomic number from `get_atom_features`

### ISSUE THREAD

**Issue by pgniewko** [pgniewko/gt-pyg#105]
**File:** `gt_pyg/data/atom_features.py:383`

```python
atom_feature_vector += [float(atomic_num)]
```

The atomic number is appended unscaled while every other slot is near unit scal.

Node features feed straight into `nn.Linear` (`gt_pyg/nn/model.py:130`) and `input_norm` runs after the projection, so the oversized column dominates the embedding at init and gets ~50x larger weight gradients than its neighbours.

The feature is also redundant. 

## Fix

Delete line 383. Keep the `atomic_num` assignment -`get_period` and `get_group` still use it.

Also drop the `- Atomic number (scalar)` bullet from the docstring (line 323).

## Impact

`node_dim_in` drops by 1 and existing checkpoints are invalidated. 
Accepted: rescaling instead of removing would break them just as hard, since the trained `node_emb` column is fit to the old 1–82 input distribution.
